src/main/java/org/gradle/profiler/GradleScenarioDefinition.java

```diff
@@ -65,7 +65,7 @@ public static String safeFileName(String name) {
 
     @Override
     public String getBuildToolDisplayName() {
-        return buildConfiguration.getGradleVersion().getVersion();
+        return buildConfiguration.getGradleVersion().toString();
```

Comment:
So the build tool for Gradle scenarios gets displayed as `Gradle 6.7`, not `6.7`.